GirafAPI/Endpoints/OrganizationEndpoints.cs

```diff
@@ -26,11 +26,11 @@ public static RouteGroupBuilder MapOrganizationEndpoints(this WebApplication app
                     }
 
                     await dbContext.Entry(user)
-                        .Collection(u => u.Organizations!).LoadAsync();
+                        .Collection(u => u.Organizations).LoadAsync();
 
                     var organizations = new List<OrganizationNameOnlyDTO>();
 
-                    if (user.Organizations is null)
+                    if (!user.Organizations.Any())
                     {
                         return Results.NotFound();
                     }
```

Comment:
I would just return an empty array instead of not found but this could also work if you prefer it. I think you can simplify the code by just removing the if check